<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataProcessWindow</class>
 <widget class="QMainWindow" name="DataProcessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1446</width>
    <height>787</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Data Process</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="system_log_textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>1011</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>630</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 1. Open Raw Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 3. Open Processed Data</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>551</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDateEdit" name="control_start_dateEdit">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="date">
      <date>
       <year>2016</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QDateEdit" name="experiment_start_dateEdit">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2016</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Control :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Experiment :</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="day_selected_checkBox">
     <property name="geometry">
      <rect>
       <x>374</x>
       <y>9</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Day selected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="white_list_smoothing_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>White List Filter
Remove Outlier</string>
     </property>
    </widget>
    <widget class="QLabel" name="current_white_list_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>White List Filter:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>691</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <widget class="QWidget" name="bee_info">
     <attribute name="title">
      <string>Bee Info</string>
     </attribute>
     <widget class="QCustomPlot" name="bee_info_widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>661</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="weather_info">
     <attribute name="title">
      <string>Weather Info</string>
     </attribute>
     <widget class="QCustomPlot" name="temp_info_widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="rh_info_widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>661</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="pressure_info_widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>661</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="in_out_info">
     <attribute name="title">
      <string>In-Out Info</string>
     </attribute>
     <widget class="QTextBrowser" name="in_out_info_textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 5. Trajectory Analysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="data_preprocessing_pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Preprocessing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 2. Preprocessing</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 4. Remove Outlier and Filter</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>651</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="distributed_area_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Distributed Area</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mdl_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>MDL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trajectory_classify_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Trajectory Classify</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="c_value_spinBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="test_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motion_pattern_filterpushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Motion Pattern Filter</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sub_group_distributed_area_pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Sub-Group Distributed Area</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="target_list_lineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1446</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Raw_Data"/>
    <addaction name="actionOpen_Processed_Data"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Sensor_Data"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_In_Out_Data"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionObject_Tracking"/>
    <addaction name="actionWhite_List"/>
   </widget>
   <widget class="QMenu" name="menuExperiment">
    <property name="title">
     <string>Experiment</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuExperiment"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Raw_Data">
   <property name="text">
    <string>Open Raw Data</string>
   </property>
  </action>
  <action name="actionOpen_Processed_Data">
   <property name="text">
    <string>Open Processed Data</string>
   </property>
  </action>
  <action name="actionObject_Tracking">
   <property name="text">
    <string>Object Tracking</string>
   </property>
  </action>
  <action name="actionOpen_Sensor_Data">
   <property name="text">
    <string>Open Sensor Data</string>
   </property>
  </action>
  <action name="actionWhite_List">
   <property name="text">
    <string>White List</string>
   </property>
  </action>
  <action name="actionOpen_In_Out_Data">
   <property name="text">
    <string>Open In-Out Data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
